main.js

```diff
@@ -1,17 +1,21 @@
 const electron = require('electron');
 const app = electron.app;
 const BrowserWindow = electron.BrowserWindow;
+const ipcMain = require('electron').ipcMain;
 
 const path = require('path');
 const url = require('url');
-const { ipcMain } = require('electron');
 
-let mainWindow;
+let welcomeWindow;
 
-function createWindow() {
+function createWelcomeWindow() {
     welcomeWindow = new BrowserWindow({
-        width: 700, height: 600,
-        frame: false, "node-integration": true
+        width: 700,
+        height: 600,
+        frame: false,
```

Comment:
frame: true

Just disable the menu bar instead. It's just annoying to deal with a frameless window since we don't have any close/minimize buttons implemented in welcome screen